plugins/zengame/includes/API/class-rest-handler.php

```
@@ -243,95 +252,172 @@ private static function get_user_rank_data(int $user_id): array
+                if (!\function_exists('gamipress_has_user_earned_achievement')) {
+                    continue;
+                }
+
                 $items = \get_posts([
-                    'post_type' => $type['slug'],
+                    'post_type' => $achievement_type,
                     'posts_per_page' => -1,
+                    'post_status' => 'publish',
                     'fields' => 'ids'
                 ]);
                 $final_items = [];
```

PHPMD (2.15.0)

[error] The variable $final_items is not named in camelCase. (undefined)

(CamelCaseVariableName)